### PR TITLE
fix pokemon search

### PR DESCRIPTION
**Pull request by JacerOmri**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "version": "0.0.1-alpha",
   "main": "main.js",
   "scripts": {
-    "postinstall": "cd ./gofbot/ && pip install -r requirements.txt",
+    "postinstall": "cd ./gofbot/ && sudo pip install -r requirements.txt",
```
Comment (Outpox) [inline package.json]: Nope nope nope. Doing this would break the Windows install.

**Comment by Outpox** [comment]
https://github.com/PokemonGoF/PokemonGo-Bot-Desktop/pull/113/commits/4eeff19b3bcdc64968dc8285b512a536f71a7831#r74373028

**Comment by JacerOmri** [comment]
@Outpox you're right, didn't mean to get it there, my mistake.

**Comment by ProjectBarks** [comment]
@JacerOmri What exactly did you change? To me it just looks like you reordered code?

**Comment by JacerOmri** [comment]
@ProjectBarks yes, the modal is created more than once, the event subscription should be fired everytime inventory modal is created.

the rest of the commit fix some errors related to map after the refactoring done in another PR (i should have another PR later tonight to fix refactored code) .

**Comment by JacerOmri** [comment]
everything has changed, closing this PR

**Pull request closed without merging**